python/ray/serve/api.py

```diff
@@ -266,6 +272,7 @@ def deployment(
     health_check_period_s: Default[float] = DEFAULT.VALUE,
     health_check_timeout_s: Default[float] = DEFAULT.VALUE,
     logging_config: Default[Union[Dict, LoggingConfig, None]] = DEFAULT.VALUE,
+    tracing_config: Default[Union[Dict, TracingConfig, None]] = DEFAULT.VALUE,
```

Comment:
please add info for this to the docstring and label it as experimental

as I mentioned, ideally we have one PR that is only adding the public API and another than adds the implementation